packages/cms-cli/README.md

```diff
@@ -96,3 +96,11 @@ Authenticate against a portal using OAuth2
 ```bash
 hs auth oauth2
 ```
+
+Upload a file or directory to the File Manager
+
+```bash
+hs filemanager upload --portal=DEV [src] [dest]
+```
+
+**Note:** Uploading to the File Manager is only supported when using the `apiKey` at this point.
```

Comment:
@williamspiro how do you feel about documenting the command before we have done the work needed to support OAuth.

Reply:
I think that is okay as it will allow us to get feedback faster  👍